src/Components/Website.tsx

```diff
@@ -0,0 +1,32 @@
+import {AbsoluteFill, spring, useCurrentFrame, useVideoConfig} from 'remotion';
+
+export const Website: React.FC<{url: string}> = ({url}) => {
```

Comment:
`WebsiteUrl` ? Or better : `Footer` ?

Reply:
Renamed it footer and changed "url" to "content" to be more coherent